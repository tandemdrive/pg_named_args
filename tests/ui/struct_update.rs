use pg_named_args::query_args;

fn main() {
    let _: (_, &[u32]) = query_args!(
        "$a, $b",
        Args {
            ..Args {
                a: "test",
                b: "test"
            }
        }
    );
}
