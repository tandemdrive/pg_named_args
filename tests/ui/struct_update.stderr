error: struct update syntax is not supported by the query_args macro
 --> tests/ui/struct_update.rs:7:13
  |
7 |             ..Args {
  |             ^^
